libraries/core/build.gradle.kts

```diff
@@ -37,6 +37,9 @@ dependencies {
     api(libs.androidx.lifecycle.common)
     api(libs.compose.animation.core)
     api(libs.compose.runtime)
+    api(libs.compose.foundation)
```

Comment:
If these aren't going to required in the future, maybe add a TODO saying to remove them before merging into main or whatever the branch is going forward.